### PR TITLE
Navigation Placement and Updating

### PR DESCRIPTION
**Pull request by Githubbubber**
Updated the social media links, fix footer bar hex color strings, update global footer placement, update component and folder names from header to topnav.

Also added a starter file dedicated to all dimensions.

Any questions, please contact me at @fssorcerer@queerglobal.com or send me a message on Slack. Thanks.

### CODE REVIEW

**Review by colinwilliams91 — APPROVED** [review]
Regarding `npm start` I've actually never had any problems with that, so I don't know if there will be an intersection, I'll report back on that.

Regarding your work in nav and other `tsx` & `css` files there are very small overlaps like the socials links (meant to get my code achieving the same thing up already to help but forgot :sweat_smile:) but I reviewed the work you did in the `nav` and it's great! Thanks for doing that. I also love the `dimensions.js` file you made, that's such a nice idea for Quality of Life.

tl;dr:
looks great, no problems on my end. thanks for doing this.
